content/elections/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 elections"
+subtitle: "Make a difference: join the OS SIG!"
+date: 2020-06-30
+draft: false
+sidebar: false # true or false to display the sidebar
+sidebarlogo: fresh-white-alt # From (static/images/logo/)
+include_footer: true
+---
+
+# Make a difference: join the OS SIG!
+
+We are excited to announce this year's Nominations for the OHBM open-science Special Interest Group (OS SIG) for OHBM 2021!
+
+Please self-nominate yourself [HERE](https://forms.office.com/Pages/ResponsePage.aspx?id=DQSIkWdsW0yxEjajBLZtrQAAAAAAAAAAAAMAAC9pqdJUN0xITDMxUktDT0RCUjJMVlc1OTVSRTlTRS4u!)
+
+The open-science movement is one of the best things happening in neuroscience. Hackathons, open-science room, new tools and methods, reproducibility and reliability, open access and inclusivity: the OS SIG is the place where it’s all happening!
+
+If open-science excites you and you want to help push its frontiers, please consider applying for a position in the OS SIG! 
+
+Early career researchers (Link to SIG), BIPOC, LGBTQ+ (Link to SIG), and people from outside Europe and North America especially encouraged to apply! 
```

Comment:
Missing links

Reply:
Student & postdoc sig: https://www.ohbmtrainees.com/
Diversity & Inclusion is a committee and doesn't have a website? But the committees are linked here: https://www.humanbrainmapping.org/i4a/pages/index.cfm?pageID=3313&activateFull=true

Are those the links you were looking for?